WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40383,918 +40424,371 @@ e_latin_empire = {
 	}
 }
 
-d_miaphysite = {
```

Comment:
From this line and onwards the file is completely messed up, making any review impossible. Discard your code and start over from scratch.